test/unit/lib.js

```diff
@@ -403,3 +403,22 @@ test('should loop through each element of an array', function() {
     }
   });
 });
+
+//getFileExtension tests
+test('should get the file extension of the passed path', function() {
```

Comment:
I think that's good. If anyone else has ideas on URL formats we should be testing for speak up.